docs/source/tutorial/tutorial-subscriptions.mdx

```diff
@@ -0,0 +1,180 @@
+2. You've created a `WebSocketTransport`, which allows the Apollo SDK to communicate with the web socket.
+3. You've created a `SplitNetworkTransport`, which can decide whether to use a web socket or not automatically, with both the `RequestChainNetworkTransport` you had previously set up, and the `WebSocketTransport` you just set up.
+4. You're now passing the `splitTransport` into the `ApolloClient`, so that it's the main transport being used in your `ApolloClient`.
+
+Now, you're ready to actually use your subscription!
+
+## Use your subscription
+
+To use the subscription you created, go to `LaunchListViewModel.swift` and start by adding a new variable to hold the subscription:
+
+```swift title="LaunchListViewModel.swift"
+@Published var launches = [LaunchListQuery.Data.Launches.Launch]()
+@Published var lastConnection: LaunchListQuery.Data.Launches?
+@Published var activeRequest: Cancellable?
+var activeSubscription: Cancellable? // highlight-line
+@Published var appAlert: AppAlert?
+@Published var notificationMessage: String?
+```
+
+Now, replace the `TODO` in the `startSubscription()` method with the following code:
+
+```swift title="LaunchListViewModel.swift"
+func startSubscription() {
+    activeSubscription = Network.shared.apollo.subscribe(subscription: TripsBookedSubscription()) { [weak self] result in
+        guard let self = self else {
+            return
+        }
+
+        switch result {
+        case .success(let graphQLResult):
+            if let tripsBooked = graphQLResult.data?.tripsBooked {
+                self.handleTripsBooked(value: tripsBooked)
+            }
+
+            if let errors = graphQLResult.errors {
+                self.appAlert = .errors(errors: errors)
+            }
+        case .failure(let error):
+            self.appAlert = .errors(errors: [error])
+        }
+    }
+}
+```
+
+Next, update the `init()` method to the following:
+
+```swift title="LaunchListViewModel.swift"
+init() {
+    startSubscription()
+}
+```
+
+Notice in `LaunchListView.swift` there is already a line to handle displaying a view from the subscription:
+
+```swift title="LaunchListView.swift"
+.notificationView(message: $viewModel.notificationMessage)
+```
+
+## Test your subscription
+
+Build and run the application, now whenever a trip is booked or cancelled (from either in the app detail view or from the Sandbox Explorer) you should see a small notification pop up at the bottom of the screen:
+
+<img alt="Subscription notification view" class="screenshot" src="images/simulator_subscription_notification_view.png" width="300"/>
+
+Congratulations, you've completed the tutorial!
+
+## More resources
+
+There are way more things you can do with the Apollo iOS SDK, and the rest of this documentation includes info on more advanced topics like:
+
+- Using [fragments](../fragments/)
+- Working with [custom scalars](../fetching-queries/#notes-on-working-with-custom-scalars)
```

Comment:
I was going to suggest adding in a step about working with custom scalars if there is a field in the schema that could support it. Mentioning it here though might be enough for now.